WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.m

```diff
@@ -364,7 +363,7 @@ - (void)configureKeyboardManager
     self.keyboardManager = [[KeyboardDismissHelper alloc] initWithParentView:self.view
                                                                   scrollView:self.tableView
                                                           dismissableControl:self.replyTextView
-                                                      bottomLayoutConstraint:self.replyTextViewBottomConstraint];
+                                                      bottomLayoutConstraint:self.keyboardManager.bottomLayoutConstraint];
```

Comment:
⚠️ I didn't know about this `KeyboardDismissHelper` earlier, but it does seem to require a `bottomLayoutConstraint`. But we cannot pass its own `self.keyboardManager.bottomLayoutConstraint` in the initializer. This can cause crashes and unexpected behavior at runtime because:
- The property `KeyboardDismissHelper.bottomLayoutConstraint` is expecting a non-optional value.
- `self.keyboardManager.bottomLayoutConstraint` is not initialized yet, and code like this is not possible in Swift. So although this is allowed in Objective-C, weird behavior can occur at Runtime.

I suggest to bring back the `self.replyTextViewBottomConstraint` property and set it to:

```objc
self.replyTextViewBottomConstraint = [self.view.keyboardLayoutGuide.topAnchor constraintEqualToAnchor:self.replyTextView.bottomAnchor];
```

Then

```objc
[NSLayoutConstraint activateConstraints:@[
  [self.replyTextView.leadingAnchor constraintEqualToAnchor:self.replyTextView.leadingAnchor],
  [self.replyTextView.trailingAnchor constraintEqualToAnchor:self.replyTextView.trailingAnchor],
  self.replyTextViewBottomConstraint
]];
```

Reply:
Makes sense! 💯 Tested this approach and it works slightly incorrectly because we reuse the same constraint in a few places. 
![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/16563318/f199a13f-17e2-4448-ba5f-7d8b9b719d9a)

Instead of that I reverted the replyTextViewBottomConstraint for the keyboard manager only.